packages/legacy/core/App/screens/PINExplainer.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react'
+/*
+import { useNavigation } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { AuthenticateStackParams, Screens } from '../../types/navigators'
+*/
+import { Trans, useTranslation } from 'react-i18next'
+import Button, { ButtonType } from '../components/buttons/Button'
+import BulletPoint from '../components/inputs/BulletPoint'
+import { testIdWithKey } from '../utils/testable'
+
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+
+import { useTheme } from '../contexts/theme'
+
+export interface PINExplainerProps {
+  continueCreatePIN: () => void
+}
+
+const Component: React.FC<PINExplainerProps> = ({ continueCreatePIN }) => {
+  const { t } = useTranslation()
+  const { ColorPallet, TextTheme, Assets } = useTheme()
+
+  const style = StyleSheet.create({
+    screenContainer: {
+      height: '100%',
+      backgroundColor: ColorPallet.brand.primaryBackground,
+      padding: 20,
+      justifyContent: 'space-between',
+    },
+    pageContainer: {
+      height: '100%',
+      justifyContent: 'space-between',
+    },
+    imageContainer: {
+      alignItems: 'center',
+      marginBottom: 30,
+    },
+  })
+
+  const imageDisplayOptions = {
+    fill: ColorPallet.notification.infoText,
+    height: 150,
+    width: 150,
+  }
+  
+  // const navigation = useNavigation<StackNavigationProp<AuthenticateStackParams>>()
+  const pressBack = () => {
+    // No handler on Back action yet
+    // navigation.navigate(Screens.Terms)
+  }
```

Comment:
Since we're going with the component approach and it's not a separate screen, this can be removed